### DISCUSSION TITLE
latest nuclei version ssh0deafault login module not working

### DISCUSSION THREAD

**Discussion by M7378** [projectdiscovery/nuclei#4755] [General]
Hi trying to use nuclei-templates/javascript/default-logins/ssh-default-logins.yaml and dosent appear to be  working with a command like this nuclei -l targetist.txt -t nuclei-templates/javascript/default-logins/ssh-default-logins.yaml  is there any activation command for bruteforce templates that iam missing or it simply dosent work well?

**Comment by olearycrew** [comment]
For a code template like that one, you need to include the `-code` tag for it to execute

**Reply by geeknik** [reply]
It's tagged as `fuzz`.

**Reply by olearycrew** [reply]
Yup - sorry I messed that up - you'll need to enable fuzz which is disabled by default

**Reply by olearycrew** [reply]
For a better answer to "why didn't this template run" we've documented all of the default exclusions here: https://docs.projectdiscovery.io/tools/nuclei/running#executing-nuclei-templates

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
